boot/seal_test.go

```diff
@@ -764,49 +842,50 @@ func (s *sealSuite) TestIsResealNeeded(c *C) {
 	err := boot.WriteBootChains(pbc, filepath.Join(dirs.SnapFDEDirUnder(rootdir), "boot-chains"), 2)
 	c.Assert(err, IsNil)
 
-	needed, _, err := boot.IsResealNeeded(pbc, rootdir, false)
+	needed, _, err := boot.IsResealNeeded(pbc, filepath.Join(dirs.SnapFDEDirUnder(rootdir), "boot-chains"), false)
```

Comment:
we should have similar tests of IsResealNeeded for the recovery-boot-chains file too, shouldn't we (unless I somehow missed this testing somewhere)?